### DISCUSSION TITLE
When i push into github it shows me the wrong user even though i have signed in with the right account

### DISCUSSION THREAD

**Discussion by ghost** [community/community#41140] [Projects and Issues] [answered]
### Select Topic Area

Product Feedback

### Body

I am having an issue every time I push code into GitHub. I have this account artenis-al (https://i.stack.imgur.com/QDRYu.png) and this is what it shows after I do a commit (https://i.stack.imgur.com/qQs0r.png) this is a name or nickname I have never used, and this shows for private repositories and public repositories as well.

After doing some research I removed the windows credentials of GitHub and vs code to make sure that the issue doesn't stand there (https://i.stack.imgur.com/rRTIs.png) it didn't resolve the issue, I have tried the command to remove all certificates and add them again, I added them again but the issue persists. for some reason every time I make a commit unless I upload directly the code from github.com it will show it with the wrong user as push.

**Comment by PythonCoderAS** [comment]
Check the username and email of who you're pushing as. For example, this is what I see: 
```
$ git config --global user.name
PythonCoderAS
$ git config --global user.email
13932583+PythonCoderAS@users.noreply.github.com
```
Also, you can check your project's username and email in a similar way.

```
# cd to your project
$ git config user.name
PythonCoderAS
$ git config user.email
13932583+PythonCoderAS@users.noreply.github.com
```

**Reply by ghost** [reply]
I have tried this as well. But the issues persists. The idea is that i found out that that account was a account created by a conpany i did a project for them. And i had pushed into their gitlab using that account.
And for some reason now every time i make a push from my PC(i will try another user) shows that old damn account. And i tried changing credentials and configuration with another username but i had no luck.

**Reply by PythonCoderAS** [reply]
Both global and in the project you're pushing from or just global?

**Reply by ghost** [reply]
From both. Even when i try with vs code version control plug in

**Reply by ghost** [reply]
In both actually. I tried them both. I tried to reinstall git again. But no joy in that troubleshooting neither.

**Reply by alitura7677** [reply]
9EO77D-GYE9D

**Reply by alitura7677** [reply]
> Check the username and email of who you're pushing as. For example, this is what I see: 
> ```
> $ git config --global user.name
> PythonCoderAS
> $ git config --global user.email
> 13932583+PythonCoderAS@users.noreply.github.com
> ```
> Also, you can check your project's username and email in a similar way.
> 
> ```
> # cd to your project
> $ git config user.name
> PythonCoderAS
> $ git config user.email
> 13932583+PythonCoderAS@users.noreply.github.com
> ```

**Reply by PythonCoderAS** [reply]
> 9EO77D-GYE9D

?

**Reply by ghost** [reply]
> 9EO77D-GYE9D

?

**Comment by jfullstackdev** [comment]
you have any other GitHub account on the same computer ? that will be problematic

**Reply by ghost** [reply]
How can i resolve it. Because i found out that there is a very old account that inused for a job some months ago. And all the pushes are going through that

**Reply by jfullstackdev** [reply]
maybe this will help 

https://stackoverflow.com/questions/3860112/multiple-github-accounts-on-the-same-computer

**Comment by ghost** [comment, marked as answer]
I solved it, I tried everything I found and I had to try a different browser and the issue got resolved. but it was a pain. thank you community for trying to help.

**Comment by montanoeuse** [comment]
had the same problem and it was caused by the user.email in git matching a different GitHub account.
fixed by updating user.email

**Reply by abdullah3kbri** [reply]
how did you do that ?

**Reply by noredact** [reply]
> how did you do that ?

Came here because the same thing happened to me...
I'm working through an online course and doing some practice in the github segment.
So, because i don't intend to share these repos, i just used a dummy email
*****@nowhere.com in my gitconfig (hiding the name i used incase it exposes someone)
On my github repo it showed some user pushed the commits and it linked to an actual github account
weird!
